internal/description/description.go

```diff
@@ -87,6 +87,7 @@ const (
 	ListProjects           = "List projects."
 	DownloadLogs           = "Download logs from a log collection job."
 	OpsManager             = "Ops Manager operations."
+	Agents                 = "Manage Ops Manager Agents."
```

Comment:
[nit] agents is not a proper name
```suggestion
	Agents                 = "Manage Ops Manager agents."
```